### ISSUE TITLE
(plotArray) plot multiple points for same timepoint 

### ISSUE THREAD

**Issue by meijer-jeroen** [bramvandijk88/cacatoo#8]
It would be useful if you could plot an array of sampled evolving parameters from the grid vs time.

**Comment by meijer-jeroen** [comment]
This should:
- allow you to draw multiple points for each time point (this.num_dps depends on values, not labels)
- only draw labels if the length of the values equals the length of the labels:

```
class Graph {
    /**
  *  The constructor function for a @Canvas object. 
  *  @param {Array} labels array of strings containing the labels for datapoints (e.g. for the legend)
  *  @param {Array} values Array of floats to plot (here plotted over time)
  * @param {Array} colours Array of colours to use for plotting
  * @param {String} title Title of the plot
  * @param {Object} opts dictionary-style list of opts to pass onto dygraphs
  */
    constructor(labels, values, colours, title, opts) {

        if (typeof window == undefined) throw "Using dygraphs with cashJS only works in browser-mode"
        this.labels = labels
        this.data = [values]
        this.title = title
        //this.num_dps = this.labels.length // number of data points for this graphs        
        this.num_dps = values.length // number of data points for this graphs        
        this.elem = document.createElement("div")
        this.elem.className = "graph-holder"      
        this.colours = []
        for (let v of colours) {
            if (v == "Time") continue            
            else if (v == undefined) this.colours.push("#000000")
            else if (v[0] + v[1] + v[2] == 765) this.colours.push("#dddddd")
            else this.colours.push(rgbToHex(v[0], v[1], v[2]))
        }

        document.body.appendChild(this.elem)
        document.getElementById("graph_holder").appendChild(this.elem)
        this.g = new Dygraph(this.elem, this.data,
            {
                title: this.title,
                showRoller: false,
                width: opts ? (opts.width != undefined ? opts.width : 500) : 500,
                height: opts ? (opts.height != undefined ? opts.height : 200) : 200,
                xlabel: this.labels[0],
                ylabel: this.labels.length == 2 ? this.labels[1] : "",
                //drawPoints: opts && opts.drawPoints || true,
                drawPoints: opts ? (opts.drawPoints ? opts.drawPoints : false) : false,
                pointSize: opts ? (opts.pointSize ? opts.pointSize : 0) : 0,
                strokePattern: opts ? (opts.strokePattern != undefined ? opts.strokePattern : null) : null,
                strokeWidth: opts ? (opts.strokeWidth != undefined ? opts.strokeWidth : 3) : 3,
                dateWindow: [0, 100],
                axisLabelFontSize: 10,
                valueRange: [0.000,],
                labels: labels.length == values.length ? this.labels: null,
                colors: this.colours

                
            });
    }
```

**Comment by bramvandijk88** [comment]
Issue resolved. Closed by @bramvandijk88 (solved in commit #77c071988901e08e09350e3b5f9bdfe48fa46250)
